src/pybind/mgr/cephadm/module.py

```diff
@@ -1490,6 +1490,22 @@ def get_hosts(self):
         """
         return list(self.inventory.all_specs())
 
+    @handle_orch_error
+    def get_gather_facts(self, hostname: Optional[str] = None):
+        # type: (str) -> List[Dict[str, Any]]
```

Comment:
Why not just define the output type annotation like this `get_gather_facts(self, hostname: Optional[str] = None) -> List[Dict[str, Any]]:`, I mean, just to be consistent with other functions.

Reply:
Done. Thanks!